test/conan/readEnv.test.ts

```diff
@@ -0,0 +1,36 @@
+
+import { execSync } from "child_process";
+import {
+    conan
+} from "../../src/utils/utils";
+import path = require("path");
+import fs = require('fs');
+
+
+jest.mock('vscode', () => ({
+    workspace: {
+        workspaceFolders: [{ uri: { fsPath: __dirname } }],
+    }
```

Comment:
You need to mock the `vscode.window.showErrorMessage()` otherwise the test will not run, due to mocked `vscode` module

Reply:
or maybe it was because the python was not found in my system 🤔